policyengine_us/parameters/gov/usda/summer_ebt/amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: SUN bucks received per eligible child each summer.
```

Comment:
Reformat all descriptions to say: "The United States Department of Agriculture provides/adds/does .... under the Summer EBT program "